### PR TITLE
Docs: Add plugin documentation

### LINKED ISSUE #3612: Integrate plugin installation UI
## Description
Integrate the plugin installation UI into the rest of the platform

## Tasks
- [x] Document plugin installation (0d) 
- [x] Link/button/component on the main screen/nav bar (0d)
- [x] Modify advanced multiselect to provide a link to plugin installation if it's empty (0d) @shreyamalviya 
- [x] Sidebar available updates indicator (0d)
- [x] Update available vs installed state hourly (0.5d)

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do?

Fixes #3612 

Add any further explanations here.

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by mssalvatore — COMMENTED** [review]
I think we should direct the user to the getting-started page somewhere in each of the setup/installation pages.

File: docs/content/usage/getting-started.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ After deploying the Monkey Island in your environment, navigate to `https://<ser
 
 On your first login, you'll be asked to create a username and password for the Monkey Island Server. [See this page for more details](../../setup/accounts-and-security).
 
+### Installing plugins
+
+Infection Monkey's various features are provided by plugins. No plugins are installed out of the box. To install plugins, click on **Plugins** in the navigation bar and install the plugins you want to use. Clicking on "Download All Safe Plugins" will download all plugins that are considered safe to use in production environments.
+
+![Plugin Installation Screen](/images/island/plugins_page/plugin_installation.PNG "Plugin Installation")
+
```
Comment (mssalvatore) [inline docs/content/usage/getting-started.md:28]: We should add brief, 1-sentence descriptions of each tab.

File: docs/content/usage/plugins/_index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Overview
+
+The Infection Monkey is extensible! You can install plugins to enhance the functionality of the Infection Monkey.
+
+Plugins page consists of three tabs:
+
+- **Available Plugins** - A table of plugins that are available for download and installation.
+- **Installed Plugins** - A table of plugins that are currently installed on the Monkey Island.
+- **Upload New Plugin** - A form to upload a new plugin to the Monkey Island. This is useful if you
```
Comment (mssalvatore) [inline docs/content/usage/plugins/_index.md:17]: How do I upgrade a plugin to a newer version?

Comment (VakarisZ) [reply docs/content/usage/plugins/_index.md]: If you don't know this, then we need to fix UX, not docs

**Review by mssalvatore — APPROVED** [review]

**Review by cakekoa — APPROVED** [review]

**Comment by mssalvatore** [comment]
Merged in edef061d6ef0aa64e236b7839573e7e90feb8e9f

**Pull request closed without merging**